skbeam/core/accumulators/binned_statistic.py

```diff
@@ -468,7 +468,7 @@ def __init__(self, shape, bins=10, range=None,
                 represented by function([]), or NaN if this returns an error.
         """
         if origin is None:
-            origin = shape[0]//2, shape[1]//2
+            origin = shape[0]/2., shape[1]/2.
```

Comment:
The doc string says that these should be ints.  Does anything this gets passed into require integers?